hail/python/test/hail/methods/test_statgen.py

```diff
@@ -135,6 +135,68 @@ def test_linreg(self):
         self.assertTrue(t1._same(t4a))
         self.assertTrue(t1._same(t4b))
 
+    def test_linreg_chained(self):
```

Comment:
I think you should also test equality of ``[[y1, y2]]`` to ``[y1, y2]`` where missing-ness is different.